docs/tutorials/start-a-private-network/4customchain.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Creating Your Private Network
+---
+Now that each participant has their own keys generated, you're ready to start your own custom chain.
+
+> Validators should not share the same keys, even for learning purposes. If two validators have the same keys, they will produce conflicting blocks, and be slashed.
+
+## Create a Chainspec
+
+Last time around, we used `--chain=local` which is a predefined "chainspec" that has Alice and Bob specified as validators along with many other useful defaults.
+
+Rather than writing our chainspec completely from scratch, we'll just make a few modifications to the the one we used before. To start we need to export the chainspec to a json file. Remember, further details about all of these commands are available by running `node-template --help`.
+
+```bash
+./target/release/node-template build-spec --chain=local > customSpec.json
+```
+
+The file we just created contains several fields, and one can learn a lot by exploring them. By far the largest field is a single hex number that is the Wasm binary of our runtime. It is part of what you built earlier when you ran the `cargo build` command. Having this information on-chain enables forkless upgrades.
+
+The portion of the file we're interested in is the Aura authorities (used for creating blocks) and the GRANDPA authorities (used for finalizing blocks). That section looks like this
+```json
+"aura": {
+  "authorities": [
+    [
+      "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
+      1
+    ],
+    [
+      "5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty",
+      1
+    ]
+  ]
+},
+"grandpa": {
+  "authorities": [
+    [
+      "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
+      1
+    ],
+    [
+      "5GoNkf6WdbxCFnPdAnYYQyCjAKPJgLNxXwPjwTh6DGg6gN3E",
+      1
+    ]
+  ]
+},
+
+```
+
+All we need to do is change the authority addresses listed (currently Alice and Bob) to our own addresses that we generated in the previous step. The sr25519 addresses go in the `aura` section, and the ed25519 addresses in the `grandpa` section. You may add as many validators as you like. For additional context, you can see how polkadot uses these keys and others in the [Polkadot wiki](https://wiki.polkadot.network/docs/en/learn-keys#session-keys).
+
+> A single person should do these steps and share the resulting file with their fellow validators.
+>
+> Because Rust -> Wasm builds aren't "reproducible", each person will get a slightly different Wasm blob which will break consensus if each participant generates the file themselves.
+
+Once the chainspec is prepared, convert it to a "raw" chainspec. The raw chainspec contains all the same information, but it contains the encoded storage keys that the node will use to reference the data in its local storage. Distributing a raw spec ensures that each node will store the data at the proper storage keys.
```

Comment:
I think this will be confusing for someone new to Substrate.
```suggestion
Once the chainspec is prepared, convert it to a "raw" chainspec. The raw chainspec is pre-processed by your node to ensure that everyone who uses it gets the same final result in their genesis block.
```

Reply:
I agree that a newcomer will not understand all the detail. But I think they'll understand, "It's better to use the raw one for some subtle reason". However a moderate uer may understand and appreciate the detail.

Reply:
I think you should strive for clarity, not just drop knowledge out of context. Also, I think we should be honest about the audience for this tutorial, which is likely someone who will not understand the detail.